sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/RedundancyInConstructorDestructorDeclaration.CSharp9.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+record RecordBase
+{
+    public RecordBase() { }
+    public RecordBase(int i) { }
```

Comment:
I don't understand the case you mention. The rule is concerned about destructors and constructors without parameters. Since `public RecordBase(string s) { }` has a parameter it will be considered compliant. Do I miss something?

Reply:
my bad :)